packages/cadl-ranch-specs/package.json

```diff
@@ -38,23 +38,23 @@
     "@azure-tools/cadl-ranch-api": "workspace:~"
   },
   "devDependencies": {
-    "@azure-tools/typespec-autorest": "0.46.0",
-    "@azure-tools/typespec-azure-resource-manager": "0.46.0",
-    "@azure-tools/typespec-client-generator-core": "0.46.0",
+    "@azure-tools/typespec-autorest": "0.47.0",
+    "@azure-tools/typespec-azure-resource-manager": "0.47.0",
+    "@azure-tools/typespec-client-generator-core": "0.47.1",
```

Comment:
wrong version? Should it be "0.47.0"?

Reply:
It's `0.47.1`: https://www.npmjs.com/package/@azure-tools/typespec-client-generator-core/v/0.47.1

Reply:
Ah, a bit strange, curious why does TCGC release another version with in several hours. /cc @tadelesh

Reply:
I think Chenjie mentioned in chat that there is a workaround (broken by new compiler).
You may need to update it in our repo too.

Reply:
Oh, seems I did not find that chat. Will update for Java.